### ISSUE TITLE
readonly users should be able to request uploads

### ISSUE THREAD

**Issue by jyoung8607** [commaai/connect#203]
C2/C3 owners who have shared their devices readonly with another user are doing it because they need remote support. Other than very simple things like UDS fingerprint lookups, these remote support needs usually require full logs to be uploaded.

Now that full logs can only be uploaded on-demand, each drive we want to look at needs to be manually uploaded, and today that can only be done by the actual device owner. This means we need new, novice users to constantly take extra actions.

~~From a privacy perspective, there's no difference between the full logs/full videos and the decimated logs/low quality videos, and readonly users can see the full versions once uploaded. The only thing we can't do is trigger the upload action.~~

It was recently pointed out the driver cam might be recorded but not uploaded, and requesting uploads for those as a readonly user would have a nonzero impact to privacy. That's information/video area that isn't already in the qlogs or qcams that a readonly user can already see. IMO it would be fine to restrict readonly users to requesting rlogs for upload, no cameras at all.

**Comment by JMPZ11** [comment]
Agreed - if it could be added to connect, or if uploads could be initiated via SSH from the device to make it more obscure.

I wonder if a solution would be to implement a readwrite role - right now it seems there is only owner and readonly. The the readwrite users could have permission to upload logs, etc.

**Comment by jyoung8607** [comment]
@adeebshihadeh per our discussion, it would be fine to implement this for rlogs only, no cameras. Will add a poll later.

**Comment by jyoung8607** [comment]
![image](https://github.com/commaai/connect/assets/46612682/e561b2b1-70e7-47e5-95eb-a9e607eac7d6)
